### ISSUE TITLE
Error occurred when executing StyleModelApply:  Sizes of tensors must match except in dimension 1

### ISSUE THREAD

**Issue by MMoneer** [cubiq/ComfyUI_Workflows#3]
Hello,
I got this error when trying to use the style model workflow.

<img width="1367" alt="brave_13012024_031" src="https://github.com/cubiq/ComfyUI_Workflows/assets/18519387/6ae5dc7b-ca80-4398-b6de-a40ee9d9a5c8">

Error occurred when executing StyleModelApply:

Sizes of tensors must match except in dimension 1. Expected size 1280 but got size 1024 for tensor number 1 in the list.

File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 154, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 84, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 77, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "D:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 923, in apply_stylemodel
cond = style_model.get_cond(clip_vision_output).flatten(start_dim=0, end_dim=1).unsqueeze(dim=0)
File "D:\ComfyUI_windows_portable\ComfyUI\comfy\sd.py", line 290, in get_cond
return self.model(input.last_hidden_state)
File "D:\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1518, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
File "D:\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1527, in _call_impl
return forward_call(*args, **kwargs)
File "D:\ComfyUI_windows_portable\ComfyUI\comfy\t2i_adapter\adapter.py", line 214, in forward
x = torch.cat([x, style_embedding], dim=1)

**Comment by Kallamamran** [comment]
I have the same error :(

**Comment by MMoneer** [comment]
> I have the same error :(

Add  --force-fp16 to run_nvidia_gpu.bat
```
.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --force-fp16
pause
```

**Comment by Kallamamran** [comment]
Still same problem unfortunately. Would you mind sharing a working workflow so I can test and see if I have missed something? 
Never mind the images. Those are just a trial and error. Have tried changing resolution of images, scalers and everything.
![image](https://github.com/cubiq/ComfyUI_Workflows/assets/25354644/0aa020d4-8e83-4e25-a120-a5c65b93cd0c)

**Comment by MMoneer** [comment]
I'm so sorry, I mixed up the topics. This is not a solution to this problem, This didn't fix it for me too. The --force-fp16 is for Dtype mismatch issue, not the tensor mismatch.
To Fix it you can download this clip vision 
https://huggingface.co/openai/clip-vit-large-patch14/resolve/main/pytorch_model.bin?download=true
It's work for me
![brave_20032024_044](https://github.com/cubiq/ComfyUI_Workflows/assets/18519387/67da8959-5ceb-46d6-8ac0-4fa2e816b13e)

**Comment by MMoneer** [comment]
You can Save it as clip-vit-large-patch14.bin instead of  pytorch_model.bin 
Also, switch between  t2iadapter_style_sd14v1.pth and coadapter-style-sd15v1.pth to figure out which one you like

**Comment by ClemCa** [comment]
@MMoneer Following your solution (large-patch14), I get a different error right in the load node: 
![image](https://github.com/cubiq/ComfyUI_Workflows/assets/26848693/101808ed-94c2-45cb-bca3-b4fc8256941b)
![image](https://github.com/cubiq/ComfyUI_Workflows/assets/26848693/d738f150-d9a3-4234-90b8-7cf30f2f1c36)
Do you have any idea as to how I can fix this?
